openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 """Maya look extractor."""
+from abc import ABC, abstractmethod
 import os
 import sys
 import json
 import tempfile
 import platform
 import contextlib
 import subprocess
+from openpype.lib.vendor_bin_utils import get_redshift_tool
```

Comment:
I think we shouldn't put it in `vendor_bin_utils` because we are not vendoring it. We should leave the code for tool detection in extractor because I think it is not currently used anywhere else

Reply:
Hey thank you, moved it back to the same file!